### DISCUSSION TITLE
[Feature] Auto Segmentation

### DISCUSSION THREAD

**Discussion by doit-ceo** [wkentaro/labelme#1126] [Ideas / Feature requests]
I'm looking for a way to have auto segmentation, i.e. by providing transparent png images for one object, then LabelMe can build polygons for that  ! also would be great if that works by command line.

Is this feature exists ?

![Selection_547](https://user-images.githubusercontent.com/8027780/82824877-d13ca000-9eb2-11ea-85ad-efbcb1b3bd13.png)

**Comment by thautwarm** [comment]
ditto.
I do want this feature in a pure python lib, just like:
https://github.com/abreheret/PixelAnnotationTool/
